admin/query/src/main/java/org/codice/ditto/replication/admin/query/sites/fields/IsRemoteManagedField.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ditto.replication.admin.query.sites.fields;
+
+import org.codice.ddf.admin.common.fields.base.scalar.BooleanField;
+
+public class IsRemoteManagedField extends BooleanField {
```

Comment:
Why not just use a BooleanField?

Reply:
I did this so that a better description for this field could be provided in the graphql schema because boolean field does not have a ctor that takes a description unfortunately.